test/dlc_tests/sanity/test_remote_function_compatibility.py

```diff
@@ -0,0 +1,40 @@
+from packaging.version import Version
+from packaging.specifiers import SpecifierSet
+import pytest
+
+from invoke import run
+from test.test_utils import get_python_version_from_image_uri
+
+
+@pytest.mark.usefixtures("sagemaker_only")
+@pytest.mark.model("N/A")
+@pytest.mark.integration("remote_function")
+def test_remote_function(image):
+    """
+    Test to check compatibility of sagemaker training images with sagemaker remote function.
+    """
+    python_version = get_python_version_from_image_uri(image).replace("py", "")
+    python_version = int(python_version)
+
+    if python_version < 37 or "training" not in image:
+        pytest.skip(
+            f"Skipping remote function compatibility test for {image}. Test only for training images with Python>3.6"
+        )
+
+    repo_name, image_tag = image.split("/")[-1].split(":")
+    container_name = f"{repo_name}-{image_tag}-remote-function-test"
+
+    run(
+        f"docker run -itd --name {container_name} "
+        f"--mount type=bind,src=$(pwd)/container_tests,target=/test"
+        f" --entrypoint='/bin/bash' "
+        f" --env SAGEMAKER_INTERNAL_IMAGE_URI={image}"
+        f"{image}",
```

Comment:
Please make the spaces consistent, by adding a space at the end of each line, or at the beginning of each line, so that formatting errors are easier to avoid.

In this case, there is a space missing between `f" --env SAGEMAKER_INTERNAL_IMAGE_URI={image}"` and `f"{image}"`, which leads to errors in running the test.

Reply:
Thanks for catching this, corrected.